sdk/hybridaks/Azure.ResourceManager.HybridContainerService/samples/Generated/Samples/Sample_HybridContainerServiceVmSkuResource.cs

```diff
@@ -21,7 +21,7 @@ public partial class Sample_HybridContainerServiceVmSkuResource
         public async Task Get_GetVMSkus()
         {
             // Generated from example definition: specification/hybridaks/resource-manager/Microsoft.HybridContainerService/stable/2024-01-01/examples/GetVmSkus.json
-            // this example is just showing the usage of "GetVmSkus" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "GetVMSkus" operation, for the dependent resources, they will have to be created separately.
```

Comment:
it's `GetVMSkus`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/hybridaks/resource-manager/Microsoft.HybridContainerService/stable/2024-01-01/provisionedClusterInstances.json#L836